modules/data-entry/src/main/frontend/src/questionnaire/Form.jsx

```diff
@@ -67,6 +72,9 @@ function Form (props) {
   let [ selectorDialogError, setSelectorDialogError ] = useState("");
   let [ changedSubject, setChangedSubject ] = useState();
   let [ loginDialogShow, setLoginDialogShow ] = useState(false);
+  let [ errorCode, setErrorCode ] = useState();
+  let [ errorMessage, setErrorMessage ] = useState("");
+  let [ showErrorDialog, setShowErrorDialog ] = useState(false);
```

Comment:
When a name starts with a verb, it suggests that it is a method, not a state.

```suggestion
  let [ errorDialogDisplayed, setErrorDialogDisplayed ] = useState(false);
```

Reply:
Done